client/features/accountPage/Account.jsx

```diff
@@ -6,8 +6,8 @@ const Account = props => (
     <div class="users-header">
       <h1 class="users-title">Streamfinder</h1>
       <input class="users-searchBar" type="text" placeholder="Search For Movies" />
-      <img class="users-homeIcon" src="../home/homeIcon.png" />
-      <img class="users-userIcon" src="../home/userIcon.png" />
+      {/* <img class="users-homeIcon" src="../home/homeIcon.png" />
```

Comment:
I'm having trouble serving up any images at all from the staticAssets folder. Is that why you commented this out?

Reply:
It causes unfulfillable HTTP requests to the server whenever this "page" is loaded. Those requests trigger the auth middleware that I was working on, and they don't actually load any images, so I just commented them out for now. I'm not sure why `staticAssets` isn't working.